source/jormungandr/jormungandr/autocomplete/elastic_search.py

```diff
@@ -218,6 +218,19 @@ def create_admin_field(geocoding):
         if not geocoding:
             return None
 
+        admin_list = geocoding.get('admin', {})
```

Comment:
I think it's nice to add a comment explain that this is to handle 2 geocode json versions